### PR TITLE
Updating de with latest strings

### PR DESCRIPTION
**Pull request by sba**

### CODE REVIEW

**Review by kenjis — APPROVED** [review]

File: Language/de/Router.php [hunk 1]
```diff
@@ -9,8 +9,10 @@
...
-// Router language settings
 return [
-    'invalidParameter'    => 'Ein Parameter stimmt nicht mit dem erwarteten Typ überein.',
-    'missingDefaultRoute' => 'Es konnte nicht bestimmt werden, was angezeigt werden soll. In der Routingdatei wurde keine Standardroute angegeben.',
+    'invalidParameter'         => 'Ein Parameter stimmt nicht mit dem erwarteten Typ überein.',
+    'missingDefaultRoute'      => 'Es konnte nicht bestimmt werden, was angezeigt werden soll. In der Routingdatei wurde keine Standardroute angegeben.',
+    'invalidDynamicController' => 'Ein dynamischer Kontroller ist aus Sicherheitsgründen nicht erlaubt. Route handler: {0}',
+    'invalidControllerName'    => 'Das Namensraumbegrenzungszeichen ist ein Backslash (\), kein Schrägstrich (/). Route handler: {0}',
 ];
+// Router language settings
```
Comment (paulbalandan) [inline Language/de/Router.php:18]: Why was this moved?

Comment (sba) [reply Language/de/Router.php:18]: ui, that was not intentional. sorry. 
I will add the comment again.

**Review by paulbalandan — APPROVED** [review]
